gateway/transport/interceptors/auth/auth.go

```diff
@@ -210,6 +213,9 @@ func (i *interceptor) StreamServerInterceptor(srv any, ss grpc.ServerStream, inf
 			return status.Errorf(codes.Unauthenticated, "invalid authentication")
 		}
 		userCtx, err := pguserauth.New().FetchUserContext(sub)
+		if userCtx.UserStatus != string(types.UserStatusActive) {
+			return status.Errorf(codes.Unauthenticated, "user is not active")
+		}
```

Comment:
Remove this, it's already being validated. When the context is not active or not found.